### DISCUSSION TITLE
Final project - click prototype

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#26] [Homework]
Post a link to your interactive clickthrough prototype, made with [Figma](https://figma.com).

The prototype should:
- show the spatial organization of content
- allow the navigation through all the different pages
- represent any unusual/specific interactions (scrolling elements, popups, etc.)

**Comment by bhornik** [comment]
Beckett, Liz, Alex:

https://www.figma.com/file/1ZuvJo6ZaH7YMSNknYyOO1/Sounds-of-Debauchery?type=design&node-id=0%3A1&mode=design&t=glMvL84HROvIzz7a-1

**Comment by Caythian** [comment]
Catherine: 

https://www.figma.com/file/nSfK2nMj0U9jwdMFFVQy7X/Final-Project?type=design&node-id=6%3A404&mode=design&t=BukZ7rkjMFCWJZKj-1

**Comment by Ambraaaliu** [comment]
Ambra&Chaerin&Robbin

https://www.figma.com/file/cltc1q9EPoTIanfwaeIx22/Debussy?type=design&node-id=0%3A1&mode=design&t=iK6DNdFYvsBibwJ3-1

**Comment by Cate-Hackett** [comment]
Mindmap: https://www.figma.com/board/ZUlc0bn9HpNydxTaCHYjQN/MINDMAP-personal-tech-archive?node-id=0-1&t=aO5ajLg1IkxXHZqV-1
Basic Wireframe: https://www.figma.com/design/Ve0XM06WTZ2spxu5vFmuog/Augmenting-Final?node-id=0-1&t=zG6iCn94091twhIs-1

**Comment by marianapazzi** [comment]
https://www.figma.com/design/FTGqkz4vD3vUT8WxvbCaXX/Tijuana-Game?node-id=0-1&t=J20aC3fwUZ3xQ0RX-1
